### DISCUSSION TITLE
Notice: invisible 0xA0 characters replaced in all files

### DISCUSSION THREAD

**Discussion by queengooborg** [mdn/mdn-community#167] [Localization]
Hey localization teams!  Just a head's up: a number of pull requests I've created have been merged to remove the 0xA0 (the non-breaking space) character with a regular space.  There had been hundreds of thousands of occurrences of this character, which were most likely created from the old wiki editor and carried over during the migration (the original English content also had this issue).  Thousands of files were touched in the process, so it is more than likely your open PRs were affected and need to be rebased.  Apologies in advance for the additional troubles!

**Comment by pransh15** [comment]
Closing this as resolved. Thanks for this! 🙏
